src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -47,6 +48,7 @@ private static void setupCommandTests(Application application) {
 
 		application.addTest("Mouse figure", new SelectMouseFigureOptionListener(application.getFreePanel(), DriverFeature.getDriverManager()));
 
+		application.addTest("CommandVisitorTest", new SelectCommandVisitorCounterListener(DriverFeature.getDriverManager()));
```

Comment:
It is not strictly a "test". The name should explain what it does, e.g. "Count subcommands".